src/seedu/addressbook/parser/Parser.java

```diff
@@ -121,7 +131,7 @@ private Command prepareAdd(String args) {
                     isPrivatePrefixPresent(matcher.group("isAddressPrivate")),
 
                     getTagsFromArgs(matcher.group("tagArguments"))
-            );
+                    );
```

Comment:
Perhaps you could just do `getTagsFromArgs(matcher.group("tagArguments")));` instead of leaving a line just for `);`